Source/Libraries/Adapters/DynamicCalculator/DynamicCalculator.cs

```diff
@@ -629,7 +653,9 @@ protected void Calculate(IReadOnlyDictionary<MeasurementKey, IMeasurement> measu
             m_expression ??= m_expressionContext.CompileDynamic(m_aliasedExpressionText);
 
             // Evaluate the expression and generate the measurement
-            HandleCalculatedValue(m_expression.Evaluate());
+            object calculatedValue = m_expression.Evaluate();
+            HandleCalculatedValue(calculatedValue);
+            RaiseVerboseMessage(m_expressionContext.Variables, calculatedValue);
```

Comment:
This is just a comment - and the difference it makes would likely be microscopic - but instead of pushing parameters to the stack, then making the functional call, then in the function checking if you should raise verbose messages, would it not be optimal to check the flag first, skipping the stack prep and function call (both here and above)?